src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIUsesFurnace.java

```diff
@@ -231,7 +231,7 @@ private IAIState accelerateFurnaces()
                 }
             }
         }
-        return getState();
+        return null;
```

Comment:
Is this even valid?
This should probably return some same default state.

Reply:
Actually, there is a significant behavior difference between null and getState. Returning a valid state here stops further execution on that tick, where returning null allows further execution. I can add comments here, though I found that documentation in AbstractEntityAIBasic

Reply:
@someaddons can probably confirm

Reply:
ye null state return means no action, not null means we traverse into that state and do not check further transitions of this state in the current tick. I thought about changing it, but back then this was needed for backwards compat due to the way our AI is structured/used.